### PR TITLE
[FEATURE] Add Welcome Page

### PR DESCRIPTION
**Pull request by KostLinux**
#8

### CODE REVIEW

File: views/statusPage/status.html
Comment (Matrix278) [inline views/statusPage/status.html:1]: I don't think folder names should be named as a camelCase, It's either words together or snake_case

File: views/mainPage/welcome.html
Comment (Matrix278) [inline views/mainPage/welcome.html:1]: I don't think folder names should be named as a camelCase, It's either words together or snake_case

File: docs/views/react.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+2. Run docker compose
+
+```
+docker-compose up
+```
+
+3. Navigate to `http://localhost:8000` for the Go App
+
+4. Navigate to localhost:5173 for React App
```
Comment (Matrix278) [inline docs/views/react.md:31]: I guess in step 4 could be done better as done in step 3 with the localhost link?
```suggestion
3. Navigate to `http://localhost:8000` for the Go App

4. Navigate to `http://localhost:5173` for React App
```

File: docs/views/react.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+```
+cd frontend
+npm install
+npm run dev
+```
+
+4. Navigate to localhost:8000 for GoBlitz app
+
+5. Navigate to localhost:5173 for React App
```
Comment (Matrix278) [inline docs/views/react.md:60]: Maybe here the same style as above with the link to localhost

File: docs/views/react.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+cp -rf dist/* ../public/
+```
+
+4. Run `make` to serve static files
+
+```
+make
+```
+
+5. Navigate to localhost:8000 to check that everything's fine
```
Comment (Matrix278) [inline docs/views/react.md:91]: Maybe here the same style as above with the link to localhost

File: docs/views/frontend_goblitz.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
+# FrontEnd development via GoBlitz
+
+## Assets
+
+Assets (images, gifs, css, js files etc) are served from public folder. The relative path is `/assets`.
+
+You can look an example of importing assets in the [Welcome Page Code](../../views/mainPage/welcome.html).
```
Comment (Matrix278) [inline docs/views/frontend_goblitz.md:7]: And here if you plan to change folder name by my comment, this file should be checked and maybe other docs as here the folder names are the same with camel case.

Comment (KostLinux) [reply docs/views/frontend_goblitz.md]: Yhea, fixed them in [7b6d62c](https://github.com/KostLinux/GoBlitz/pull/10/commits/7b6d62cbeeca37deb2e44f942598fe2b9ca6b21a)

**Comment by KostLinux** [comment]
Everything fixed in [7b6d62c](https://github.com/KostLinux/GoBlitz/pull/10/commits/7b6d62cbeeca37deb2e44f942598fe2b9ca6b21a)

**Pull request merged**